test/before-tests.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+const util = require('util')
+const fs = require('fs')
+const path = require('path')
+const pem = require('pem');
+
+(async () => {
+  const createCertificate = util.promisify(pem.createCertificate)
+  const keys = await createCertificate({ days: 1, selfSigned: true })
+  if (!fs.existsSync(path.join(__dirname, 'https', 'fastify.cert'))) fs.writeFileSync(path.join(__dirname, 'https', 'fastify.cert'), keys.certificate)
```

Comment:
```suggestion
  const certFile = path.join(__dirname, 'https', 'fastify.cert')
  if (!fs.existsSync(certFile)) fs.writeFileSync(certFile, keys.certificate)
```

Reply:
Can you please use the promise version instead of the Sync ones?